Source/WebKit/UIProcess/GPU/GPUProcessProxy.cpp

```diff
@@ -165,11 +161,13 @@ GPUProcessProxy::GPUProcessProxy()
     parameters.parentPID = getCurrentProcessID();
 
 #if USE(SANDBOX_EXTENSIONS_FOR_CACHE_AND_TEMP_DIRECTORY_ACCESS)
-    bool isExtension = false;
 #if USE(EXTENSIONKIT)
-    isExtension = !!extensionProcess();
+    auto cacheDirectory = "/Library/Caches/com.apple.WebKit.GPUExtension/"_s;
```

Comment:
I don't really understand why we need a separate directory for GPUExtension versus GPU process. Are we doing this because we want to be able to toggle between the two without impacting the cached state of each mode?